ChangeLog

```diff
@@ -26,6 +26,10 @@ Release date: Undefined
 
   Closes #4301
 
+* Fix crash when checking ``setup.cfg`` for pylint config when there are non-ascii characters in there
+
+  Closes #4328
```

Comment:
Sorry for not noticing it earlier, I just saw that this should belong to `2.7.5` not `2.8.0`.
Would you like to change it if not I can push a commit as well.

Reply:
Done.